### DISCUSSION TITLE
一些经典的算法与模板--笔试用 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#18] [Announcements]
# 一些经典的算法与模板--笔试用 - 没有秃头基因的燊的博客

很久没有写博客了，最近也是换了电脑，也就把博客迁移过来顺便写一篇博客吧 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/8026/

**Comment by HCY-ASLEEP** [comment]
🐮
